packages/cozy-scanner/src/DocumentCategory.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+jest.mock('cozy-ui/transpiled/react/utils/color', () => ({
+  getCssVariableValue: () => '#fff'
+}))
+import DocumentCategory from './DocumentCategory'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+
+describe('DocumentCategory', () => {
+  it('should match snapshot if selected and icon', () => {
+    const onSelect = jest.fn()
+    const { queryByText, getByText, asFragment } = render(
+      <MuiCozyTheme>
+        <DocumentCategory
+          onSelect={onSelect}
+          category={{
+            icon: 'test',
+            label: 'test'
+          }}
+          isSelected={true}
+          selectedItem={{ label: 'test', id: '1' }}
+          items={[
+            {
+              id: '1',
+              label: 'Label1'
+            },
+            {
+              id: '2',
+              label: 'Label2'
+            }
+          ]}
+          t={text => text}
+        />
+      </MuiCozyTheme>
+    )
+    expect(asFragment()).toMatchSnapshot()
+    //Click on the item
+    fireEvent.click(getByText('Scan.items.test'))
+    expect(asFragment()).toMatchSnapshot()
+    //Check if the second item is displayed in the ActionMenu
+    expect(getByText('Scan.items.Label2'))
```

Comment:
Seems like you do not expect anything here.

Reply:
No. I expect to see the ActionMenu. And to to that, I check if I see this label